v0/destinations/vero/transform.js

```diff
@@ -0,0 +1,211 @@
+const set = require("set-value");
+const get = require("get-value");
+const {
+  defaultRequestConfig,
+  CustomError,
+  constructPayload,
+  isDefinedAndNotNull,
+  getIntegrationsObj,
+  removeUndefinedAndNullValues,
+  isDefinedAndNotNullAndNotEmpty,
+  getErrorRespEvents,
+  getSuccessRespEvents
+} = require("../../util");
+const { EventType } = require("../../../constants");
+const { CONFIG_CATEGORIES, MAPPING_CONFIG } = require("./config");
+
+// This function handles the common response payload for the supported calls
+const commonResponseBuilder = (message, category, destination) => {
```

Comment:
generallly we call this func responseBuilderSimple just a suggestion to keep naming similar accross all dests (non blocking comment)

Reply:
Updated